install.sh

```diff
@@ -191,6 +191,10 @@ exists_but_not_writable() {
   [[ -e "$1" ]] && ! [[ -r "$1" && -w "$1" && -x "$1" ]]
 }
 
+exists_but_not_group_writable() {
+  [[ -e "$1" ]] && [[ "$(get_permission "$1")" != "775" ]]
```

Comment:
```suggestion
  [[ -e "$1" ]] && [[ "$(get_permission "$1")" != "2775" ]]
```
See https://github.com/Homebrew/install/pull/328#issuecomment-758859007

Reply:
Note that this comment is intended for Linux. I just tested on macOS, and the _default_ behaviour on macOS is the same as `2775` directories on Linux! 🤯

Reply:
I don't know what bit `02000` (`S_ISGID`) does on Darwin.

Reply:
> Note that this comment is intended for Linux. I just tested on macOS, and the _default_ behaviour on macOS is the same as `2775` directories on Linux!

That turns out to be [true on most (all?) BSDs](https://docstore.mik.ua/orelly/unix/upt/ch22_05.htm): SysV-based OSes tend to respect the effective GID of the creator, while BSDs tend to respect the parent dir's group.

As for macOS...

`mkdir(2)`:
> The directory's owner ID is set to the process's effective user ID.  The directory's group ID is set to that of the parent directory in which it is created.

`open(2)`:
> When a new file is created, it is given the group of the directory which contains it.

Reply:
> I don't know what bit `02000` (`S_ISGID`) does on Darwin.

Can't find any documentation on that either.

Reply:
It's probably there just for compatibility.